packages/block-editor/src/components/block-draggable/draggable-chip.js

```diff
@@ -40,10 +41,16 @@ export default function BlockDraggableChip( { clientIds } ) {
 					<FlexItem>
 						<BlockIcon icon={ handle } />
 					</FlexItem>
-					<FlexItem>
-						<BlockIcon icon={ icon } />
-					</FlexItem>
-					<FlexItem>{ `(${ clientIds.length })` }</FlexItem>
+					{ clientIds.length === 1 && (
```

Comment:
This also removes the icon for multiple blocks of the same type. Is that intentional? Maybe the icon is still useful in this case as it would mean 3 blocks of the type "paragraph".

Reply:
Actually intentional, yes. In the beginning of this PR I kept the behavior, with even the stack icon showing for multiple blocks. But as I added the "blocks" text, I realized that the additional icon is just redundant and doesn't add a ton of value once you drag more than one.